### PR TITLE
Add Topic and Topic List

### LINKED ISSUE #4: Create DiscussionItem component

### PR DESCRIPTION
**Pull request by cdsandoval**
Close #4 #5 
- Add Topic Item to Main
- Add Topic List to Main

### CODE REVIEW

File: src/components/TopicList.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
+import Topic from "./Topic";
+
+const topics = [
```
Comment (valevassallo) [inline src/components/TopicList.jsx]: This is something you'll get from the localStorage afterwards, right?

Comment (cdsandoval) [reply src/components/TopicList.jsx]: Yes, this a fake date to render something in the body.

Comment (valevassallo) [reply src/components/TopicList.jsx]: okay, cool

Comment (byteconde) [reply src/components/TopicList.jsx]: Hey guys, I would like to share with you a better way to organize your folders. [Click here](https://medium.com/@alexmngn/how-to-better-organize-your-react-applications-2fd3ea1920f1)

Comment (valevassallo) [reply src/components/TopicList.jsx]: Thanks

Comment (valevassallo) [reply src/components/TopicList.jsx]: Thanks @byteconde

Comment (valevassallo) [reply src/components/TopicList.jsx]: Thanks @byteconde

Comment (valevassallo) [reply src/components/TopicList.jsx]: Thanks @byteconde

**Pull request merged**